Entity/AddressEmbeddable.php

```diff
@@ -17,7 +18,17 @@ class AddressEmbeddable extends \CommerceGuys\Addressing\Model\Address
     /**
      * @ORM\Column(type="string", nullable=true)
      */
-    protected $recipient;
```

Comment:
Removing recipient that's no longer in the model seems like a good call, but the getter and setter should be removed too, and additional getter and setters should be created for $givenName, $additionalName and $familyName

Reply:
Changes look good